src/content/docs/guides/game design/godot/basics.md

```diff
@@ -0,0 +1,182 @@
+![Drop-down of shape property on CollisionShape2D node highlighting New CircleShape2D](./CircleShape.png)
+
+Similarly, on the **Sprite2D** node, you can change the texture property to a new texture. Save the little birdie image below to your device with **right-click > Save Image As...** and drag it into the FileSystem tab in the bottom left of the screen, alternatively you can make your own image and use that. Once it's in the FileSystem tab, you can drag it into the texture property of the Sprite2D node. 
+![Cute pixel art circular sprite](./LilBirdie.png)
+If you want to make your own pixel art similar to this to expand your game later, use the [PICO-8 palette](https://lospec.com/palette-list/pico-8).
+
+Your scene should now look something like this:
+![Scene with tiny texture in the viewport](./UpdatedScene.png)
+
+In the viewport is the largest area of the screen that shows the scene. You can move around the viewport with the middle mouse button and zoom in and out with the scroll wheel.
+Zooming into the small circle where the red and green lines meet, you might notice the pixelated texture is fuzzy. This is because the texture is being scaled up to fit the size of the sprite. To fix this, go to the Sprite2D node and change the property **CanvasItem > Texture > Filter** to **Nearest**. Only do this for pixel art textures.
+In the future, if you do not want to change the Filter property for every Sprite2D node, you can change any parent node's Filter property to Nearest and all child nodes will inherit it if their Filter property is set to Inherit. For an all pixel art game you might as well set the root node's (**Node2D**) Filter property to Nearest.
+
+### Scenes
+Scenes are collections of nodes that make up a part of your game. They can be anything from a single node to a complex hierarchy of nodes. Scenes can be saved as files and reused in other scenes.
+
+For example, it's good practice to have your player character in its own scene, so you can reuse it in other scenes. This is especially useful if you have multiple levels in your game, you can reuse the player character scene in each level scene. You can convert the current CharacterBody2D node into a scene by right-clicking it and selecting **Save Branch as Scene**. Save it as **player.tscn**, also rename the node from **CharacterBody2D** to **player** by double-clicking the name. Now if you want to edit the child nodes of your player character, you can open the player.tscn scene by clicking the new **Open in Editor** button next to the player node in the Scene tab. You can also open the scene by double-clicking the player.tscn file in the FileSystem tab.
```

Comment:
This should be a note

Reply:
Can't do notes on .md files, I remember you saying to use .mdx only for code heavy files, so I was just working with what I had

Reply:
That's annoying